server/src/test/scala/za/co/absa/atum/server/api/controller/PartitioningControllerSpec.scala

```diff
@@ -56,17 +56,17 @@ class PartitioningControllerSpec extends ZIOSpecDefault with TestData {
           for {
             result <- PartitioningController.createPartitioningIfNotExists(partitioningSubmitDTO1)
           } yield assertTrue(result == expectedAtumContextDTO)
-        },
+        } @@ TestAspect.tag("IntegrationTest"),
```

Comment:
The tag can be applied on an entire suite or spec as a whole, there is no need to tag every individual test.

Reply:
Addressed.